apps/tutorial_servo_can_control/src/main.cpp

```diff
@@ -0,0 +1,20 @@
+#include "CANBuffer.h"
+#include "CANBus.h"
+#include "CANMsg.h"
+#include "hw_bridge.h"
+#include "mbed.h"
+#include "TutorialServo.h"
+
+PwmOut servoPwmOut(PA_1);
+
+CANBus can1(CAN1_RX, CAN1_TX, HWBRIDGE::ROVERCONFIG::ROVER_CANBUS_FREQUENCY);
+
+int main() {
+  CANMsg rxMsg;
+  TutorialServo servo(servoPwmOut);
+  while (true) {
+    can1.read(rxMsg)
+    servo.setPositionInDegrees(rxMsg)
```

Comment:
Use CANMsg's getPayload() function to extract the msg content. See the arm app for examples